solutions/is_leap_year.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for determining whether the given year is leap year or not
+
+Module contents:
+    - is_leap: determines whether the given year is leap year or not.
+
+Created on XX XX XX
+@author: Khusro Sakhi
+"""
+
+
+def is_leap_year(year) -> bool:
```

Comment:
you forgot inputs type annotations

Reply:
You're right I happen to have missed it 😅.